testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = LazySettings(
+    envvar_prefix="TesTFM",
+    core_loaders=["YAML"],
+    settings_file=["settings.yaml"],
+    preload=["*.yaml"],
+    includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
```

Comment:
Will you have secrets defined here ? Or the plan is to have them in OCP ?

If in OCP, please remove them from here.